frontend/src/components/widgets/inputs/MenuBarInput.vue

```diff
@@ -36,6 +36,9 @@
 			align-items: center;
 			white-space: nowrap;
 			padding: 0 8px;
+			background: none;
```

Comment:
Are these three added lines doing something? It seems like they are already the state. But I haven't tested/checked.

Reply:
The `<button>` tag has a background and whitespace css.

Reply:
Oh right, forgot you changed it to a button!